inventory-repository/src/main/resources/ldap/inventory.schema

```diff
@@ -821,42 +821,47 @@ objectclass ( InventoryObjectClasses:3
   SUP top
   AUXILIARY )
 
-objectclass ( InventoryObjectClasses:80:1
+objectclass ( InventoryObjectClasses:80.1
```

Comment:
ok, after confirmation from @fanf , this information is irrelevant, and so i'm merging it